pulp_file/tests/functional/conftest.py

```diff
@@ -66,6 +66,17 @@ def file_random_content_unit(
     return gen_object_with_cleanup(file_content_api_client, **artifact_attrs)
 
 
+@pytest.fixture
```

Comment:
```suggestion
def file_content_unit_with_name_factory(
```

Anytime the factory pattern is used we've started adding `_factory` to the end to clearly indicate it's usage as a factory.